babel.config.js

```diff
@@ -1,9 +1,10 @@
 module.exports = {
   presets: [
     '@babel/preset-typescript',
-    '@babel/preset-react',
```

Comment:
I wonder if this change will affect the output generated in the `build` folder and hence what's get packaged into NPM package. How would that affect our users.

Reply:
We are shipping some minimal JSX e.g. in `detectHostComponents`.